main.go

```diff
@@ -61,9 +66,17 @@ func main() {
 	kubeInformerFactory := kubeinformers.NewSharedInformerFactory(kubeClient, time.Second*30)
```

Comment:
> I think you want a construction like this: https://github.com/openshift/service-serving-cert-signer/blob/master/pkg/operator/starter.go#L26 so you're properly scoped, right?

this can be a followup if you like.